crates/apub/src/protocol/objects/group.rs

```diff
@@ -110,7 +110,7 @@ impl Group {
       deleted: Some(false),
       nsfw: Some(self.sensitive.unwrap_or(false)),
       actor_id: Some(self.id.into()),
-      local: Some(false),
+      local: None,
```

Comment:
Actually this line shouldnt be changed, please revert it. Changing the line below is enough to fix the problem.

Reply:
Changed in [87e31d3](https://github.com/LemmyNet/lemmy/pull/3088/commits/87e31d3979cf1f46b9dd37f1ce7071ad698508ee)